content/blog/2024-04-30-dkg-for-multi-sig.mdx

```diff
@@ -0,0 +1,51 @@
+---
+author: iflabs
+title: "Announcing Distributed Key Generation for multisig"
+description: "Iron Fish multisignature wallets now supports Distrubuted Key Generation (DKG) in addition to Trusted Dealer Key Generation (TDK)."
+image: /images/blog/multisig.png
+tags: [ironfish, iron fish, multi-sig, multi sig, multi-sig wallets]
+---
+
+Earlier this month we [announced](2024-04-09-multi-sig.mdx) support for multisignature transactions, or multisig, which allow you to spread the key required to approve a transaction among multiple participants. Need a refresher on what multisignature transactions are? Check out the previous [blog post](/learn/blog/2024-04-09-multi-sig)!
```

Comment:
The first link doesn't work on the preview, but as a separate question, do we need both?

Reply:
Removed the first one!